Sofa/Component/Mapping/Linear/src/sofa/component/mapping/linear/BarycentricMappers/BarycentricMapperTopologyContainer.h

```diff
@@ -148,7 +148,8 @@ class BarycentricMapperTopologyContainer : public TopologyBarycentricMapper<In,O
     ~BarycentricMapperTopologyContainer() override = default;
 
     virtual type::vector<Element> getElements()=0;
-    virtual type::vector<SReal> getBaryCoef(const Real* f)=0;
+    virtual std::array<SReal, Element::NumberOfNodes> getBarycentricCoefficients(const Real* f)=0;
```

Comment:
```suggestion
    virtual std::array<Real, Element::NumberOfNodes> getBarycentricCoefficients(const Real* f)=0;
```
Is it ok to return `Real` and not `SReal` here?

Reply:
Could you add a description of the `f` parameter? Or maybe a more explicit name?

Reply:
> Is it ok to return `Real` and not `SReal` here?

The main reason is that I took the same type as the getBaryCoef 😅.
I guess it would be better to set the same type everywhere